ios/manifest

```diff
@@ -2,18 +2,18 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 3.0.0
+version: 3.0.1
```

Comment:
Please do a major bump to `4.0.0`

Reply:
I use this tool to build for all modules I recompile xcframeworks... works good...

https://github.com/jeffctown/xcframework